Userland/Utilities/beep.cpp

```diff
@@ -4,11 +4,16 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
-ErrorOr<int> serenity_main(Main::Arguments)
+ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    TRY(Core::System::beep());
+    Optional<size_t> tone;
+    Core::ArgsParser args_parser;
+    args_parser.add_option(tone, "Beep tone", "beep-tone", 'n', "Beep tone (frequency in Hz)");
```

Comment:
The `beep` utility on Linux uses `-f` for setting the frequency. Seems like BSD does too? So IMO it'd be better to use that for consistency.

Reply:
Done :)